### PR TITLE
Expect None in some values when converting to str a ctfModel. There a…

### PR DESCRIPTION
**Pull request by pconesa**
…re already cases where resolution a fit quality are missing from ctf refinement protocols. It might be a bug, but this tolerates future bugs.

### CODE REVIEW

**Comment by azazellochg** [comment]
@pconesa what are "ctf refinement protocols"?

**Comment by pconesa** [comment]
Need to check that but I believe one of the particle CTF refinement one. Thera was a gCTF and I believe Xmipp has one too.

**Comment by pconesa** [comment]
Done!

File: pwem/objects/data.py [hunk 1]
```diff
@@ -136,14 +136,18 @@ def __init__(self, **kwargs):
 
     def __str__(self):
         phaseShift = self.getPhaseShift() if self.hasPhaseShift() else 0
```
Comment (delarosatrevin) [inline pwem/objects/data.py:138]: I think now you can treat phaseShift the same as the other ones.

**Review by delarosatrevin — APPROVED** [review]
Nice @pconesa . Just a minor comment left.

File: pwem/objects/data.py (hunk 1 above)
Comment (pconesa) [reply pwem/objects/data.py]: Yep, done.

**Pull request merged**